packages/protocol/test-sol/governance/network/Governance.t.sol

```diff
@@ -684,14 +686,14 @@ contract SetConstitution is BaseTest {
     governance.setConstitution(address(governance), 0x00000000, threshold);
   }
 
-  function testRevertsIf_ThresholdIsNotGreatedThanMajority() public {
```

Comment:
According to the naming convention, isn't this supposed to be `test_Revert[If|When]_Condition`